js/modules/k6/grpc/client.go

```diff
@@ -461,7 +461,11 @@ func (c *Client) parseInvokeParams(paramsVal goja.Value) (*invokeParams, error)
 				// TODO(rogchap): Should we manage a string slice?
 				strval, ok := kv.(string)
 				if !ok {
-					return result, fmt.Errorf("metadata %q value must be a string", hk)
+					binval, ok := kv.([]byte)
```

Comment:
Checking the https://grpc.io/docs/what-is-grpc/core-concepts/#metadata, it seems like we should only treat the value as binary only for the cases, where keys have a postfix `-bin`.

What do you think about considering this?